core/drivers/crypto/crypto_api/acipher/ecc.c

```diff
@@ -107,6 +107,8 @@ static TEE_Result ecc_generate_keypair(struct ecc_keypair *key,
 	}
 
 	key_size_bits = get_ecc_key_size_bytes(key->curve) * 8;
```

Comment:
Wouldn't it be more convenient with a `get_ecc_key_size_bits()` which does the right thing?